scripts/base.sh

```diff
@@ -88,16 +88,18 @@ function parse_arguments {
 }
 
 
-# ---------------------------------------- 
+# ----------------------------------------
 # start watching rds log files and write it
 # to files so rsyslog can monitor then
-# ---------------------------------------- 
+# ----------------------------------------
 function configure_rds() {
     mkdir -p /root/.aws
     echo "[default]" > $AWS_CREDENTIAL_FILE
     echo "region = $AWS_REGION" >> $AWS_CREDENTIAL_FILE
-    echo "aws_secret_access_key = $AWS_SECRET_KEY" >> $AWS_CREDENTIAL_FILE
-    echo "aws_access_key_id = $AWS_ACCESS_KEY" >> $AWS_CREDENTIAL_FILE
+    if [[ ! -z $AWS_SECRET_KEY ]]; then
+      echo "aws_secret_access_key = $AWS_SECRET_KEY" >> $AWS_CREDENTIAL_FILE
+      echo "aws_access_key_id = $AWS_ACCESS_KEY" >> $AWS_CREDENTIAL_FILE
+    fi
```

Comment:
please add a log if we are depending on IAM role

Reply:
done